src/support/azext_support/custom.py

```diff
@@ -272,3 +280,102 @@ def pre_operations(self):
         super().pre_operations()
         args = self.ctx.args
         _check_name_availability_no_subscription(self.cli_ctx, str(args.communication_name), "Microsoft.Support/communications")
+
+class FileWorkspaceCreateNoSubscription(_CreateNoSubscriptionFileWorkspace):
+    def pre_operations(self):
+        from azext_support._validators import _check_name_availability_no_subscription
+        super().pre_operations()
+        args = self.ctx.args
+        file_workspace_name = str(args.file_workspace_name)
+        _check_name_availability_no_subscription(self.cli_ctx, file_workspace_name, "Microsoft.Support/workspace")
```

Comment:
I dont see an addition to the enums in Check Name Availability for this resource type